src/functions.js

```diff
@@ -1,24 +1,34 @@
+// @flow
+/** Include this to ensure that all functions are defined. */
 import utils from "./utils";
 import ParseError from "./ParseError";
 import ParseNode from "./ParseNode";
-import {default as _defineFunction, ordargument} from "./defineFunction";
+import {
+    default as _defineFunction,
+    ordargument,
+    _functions,
+} from "./defineFunction";
 
-// WARNING: New functions should be added to src/functions.
+import type {FunctionPropSpec, FunctionHandler} from "./defineFunction" ;
+
+// WARNING: New functions should be added to src/functions and imported from
+// here.
```

Comment:
`here => there`

Reply:
Sorry, my meaning didn't come out clearly. I changed it to "imported here". The point is that whoever imports `functions.js` should be guaranteed that all definitions are then available; and that's assured only by `functions.js` importing all the definitions.

I am open to other rewordings.